content/09.evolution.md

```diff
@@ -96,11 +96,45 @@ IFNs and Ribavirin have shown promise in _in vitro_ analyses of MERS, but their
 Therefore, only limited strategies can be adopted for the pharmaceutical management of COVID-19 from previous severe HCoV infections.
 Research in response to prior outbreaks of HCoV-borne infections, such as SARS and MERS, have, however, provided a strong foundation for hypotheses about the pathogenesis of SARS-CoV-2 as well as potential diagnostic and therapeutic approaches.
 
-### Zoonotic Transfer of Coronaviruses
-
-<!-- TO DO: How does zoonotic transfer happen? What role has this played in previous HCoV outbreaks? How do viruses evolve during host species changes?-->
-
-If not addressed, economic and environmental stressors are likely to cause future zoonotic transfer of diseases in the future [@doi:10.1126/science.abc3189].
+### Question 2: Where Did It Come From?
+
+Despite the high degree of similarity to SARS-CoV-1, even greater sequence identity was observed between SARS-CoV-2 and zoonotic coronaviruses.
+A 2001 literature review estimated that 61% of human pathogens have a zoonotic origin [@doi:10.1098/rstb.2001.0888].
+A zoonotic disease, or zoonosis, arises when a pathogen can both a) infect and b) cause a disease in humans [@doi:10.1016/j.pt.2016.04.007].
+As a result, the risk of zoonotic disease increases when there is substantial interaction between humans and wildlife [@doi:10.1016/j.pt.2016.04.007].
+Many factors can influence this human/wildlife interface and therefore the risk of zoonotic transmission events [@doi:10.1016/j.pt.2016.04.007; @doi:10.3201/eid1112.040789].
+
+In the SARS epidemic, SARS-CoV-1 was also thought to have emerged in a live animal market.
+A survey of a market in Shenzen, China revealed that individuals from two carnivore species, namely several masked palm civets (_Paguma larvata_) and one racoon dog (_Nyctereutes procyonoides_), were likely carriers of SARS-CoV-1, despite presenting as healthy [@doi:10.1126/science.1087139].
+However, further analysis suggested that these species might be only intermediate hosts who were exposed in the market setting [@doi:10.7150/ijbs.45472].
+A closely related virus was identified in Chinese horseshoe bats (_Rhinolophus sinicus_), but the sequence identity was only 88% with SARS-CoV-1 [@doi:10.1073/pnas.0506735102].
+Therefore, the species of origin for SARS-CoV-1 remains unresolved.
+
+In the case of SARS-CoV-2, early interest for the emergence of the pathogen turned to live-animal markets in Wuhan [@doi:10.1038/d41586-020-00548-w; @url:https://www.thestandard.com.hk/sections-news-print/215457/Ill-husband-and-wife--add-to-Wuhan-riddle].
```

Comment:
```suggestion
In the case of SARS-CoV-2, early interest for the emergence of the pathogen turned to live-animal markets in Wuhan [@doi:10.1038/d41586-020-00548-w; @url:https://www.thestandard.com.hk/sections-news-print/215457/Ill-husband-and-wife--add-to-Wuhan-riddle], where it would later emerge that many animals were sold suffering from poor health and hygiene [@doi:10.1038/s41598-021-91470-2].
```